signer/src/stacks/api.rs

```diff
@@ -933,7 +948,7 @@ impl StacksInteract for StacksClient {
             FeePriority::High => resp.estimations.last().map(|est| est.fee),
         };
 
-        Ok(fee_estimate.unwrap_or(DEFAULT_TX_FEE).min(MAX_TX_FEE))
+        Ok(fee_estimate.unwrap_or(default_min_fee))
```

Comment:
Hmm, well I'm not sure this will solve the issue. I think the issue is that fee_estimate is `Some(X)` but `X < default_min_fee`.

Also, this no longer constrains the maximum fee that a signer will pay. That was added in case someone's stacks node returns some wonky estimate that is probably incorrect. No idea if it is necessary though.

Reply:
> Hmm, well I'm not sure this will solve the issue. I think the issue is that fee_estimate is Some(X) but X < default_min_fee.

Ah okay, changed it now so it solves both stacks-node errors (i.e. if it returns that it can't provide a fee) and if the fee is too small (pending the change for the "transaction" above). We should still "officially" decide on a multiplier though.

> Also, this no longer constrains the maximum fee that a signer will pay. That was added in case someone's stacks node returns some wonky estimate that is probably incorrect. No idea if it is necessary though.

Ahh right whoops, my brain short-circuited there, wasn't intended to remove that :sweat_smile: